AltBeacon.Library/src/AltBeacon/Beacon/BeaconParser.cs

```diff
@@ -416,8 +417,7 @@ public BeaconParser SetBeaconLayout(string beaconLayout)
 
                 if (!found)
                 {
-                    ////TODO LogManager
-                    ////LogManager.d(Tag, "cannot parse term " + term);
+                    Logger.Debug("cannot parse term " + term);
```

Comment:
Hmm, then the flag name "VerboseLoggingEnabled" must be changed to "BeaconParserLoggingEnabled", right ?